configs/estore.json

```diff
@@ -102,6 +102,90 @@
         }
       }
     },
+    "estore-pro-furniture": {
+      "isPro": true,
+      "title": "eStore Pro Furniture",
+      "preview": "https://themegrilldemos.com/estore-pro-furniture/",
+      "description": "This modern furniture WordPress template is packed with elements and features you ever need.",
+      "template": [
+        "estore-pro"
+      ],
+      "category": [
+        "all",
+        "others"
+      ],
+      "plugins_list": {
+        "everest-forms": {
+          "name": "Everest Forms",
+          "slug": "everest-forms/everest-forms.php"
+        },
+        "woocommerce": {
+          "name": "WooCommerce",
+          "slug": "woocommerce/woocommerce.php"
+        },
+        "yith-woocommerce-wishlist": {
+          "name": "YITH WooCommerce Wishlist",
+          "slug": "yith-woocommerce-wishlist/init.php"
+        }
+      },
+      "core_options": {
+        "blogname": "eStore Pro Furniture",
+        "page_on_front": "eStore Home",
+        "page_for_posts": "Blog"
+      },
+      "widgets_data_update": {
+        "dropdown_categories": {
+          "category": {
+            "estore_featured_posts_carousel_widget": {
+              "1": {
+                "category": "Uncategorized"
+              }
+            }
+          },
+          "product_cat": {
+            "estore_woocommerce_product_grid": {
+              "1": {
+                "category": "Trendy Products"
+              }
+            },
+            "estore_woocommerce_product_slider": {
```

Comment:
Please check if the widget of this ID is used or not. While checking on there are four TG Product Carousel widgets and one TG Product grid widget.